pkgs/development/tools/github-changelog-generator/default.nix

```diff
@@ -0,0 +1,18 @@
+{ stdenv, lib, bundlerEnv, ruby}:
+bundlerEnv rec {
+  name = "github-changelog-generator-${version}";
+  version = "v1.14.3";
+  gemdir = ./.;
+
+  postBuild = ''
+    rm $out/bin/{bundle,bundler,ghclgen,rake}
+    '';
```

Comment:
You can use `bundlerApp`, which will take care of this for you.

Reply:
thanks for the tipp :)